plugins/tech-insights-backend/api-report.md

```diff
@@ -106,6 +111,41 @@ export type TechInsightsContext<
   factRetrieverEngine: FactRetrieverEngine;
 };
 
+// @public
+export class TechInsightsDatabase implements TechInsightsStore {
+  constructor(db: Knex, logger: Logger);
```

Comment:
If the `TechInsightsStore` needs to be exported I would opt for a private constructor and an async static `create()` method which accepts options to future proof a bit more.

Reply:
Leaving this untouched and package internal instead of exposing it.